### PR TITLE
Delete email info

### PR DESCRIPTION
**Pull request by rindibudiaramdhan**
v1.2.7

There is a need to delete email info in email footer.

image

### CODE REVIEW

**Review by ayocodingit — APPROVED** [review]

File: tests/Feature/LogisticRequestTest.php [hunk 1]
```diff
@@ -274,20 +274,21 @@ public function testPostRequestApproval()
...
+    //     Notification::fake();
+
+    //     $response = $this->actingAs($this->admin, 'api')->json('POST', '/api/v1/logistic-request/final', [
+    //         'agency_id' => $this->applicant->id,
+    //         'applicant_id' => $this->applicant->agency_id,
+    //         'approval_status' => ApplicantStatusEnum::approved(),
+    //         'url' => 'http:://localhost/#',
+    //     ]);
+    //     $response->assertSuccessful();
+    // }
```
Comment (rachadiannovansyah) [inline tests/Feature/LogisticRequestTest.php:291]: is this still used sir?

Comment (rindibudiaramdhan) [reply tests/Feature/LogisticRequestTest.php:291]: yes. i note it in the comments 😄 
lol sir 🤣

**Review by firmanJS — APPROVED** [review]

File: tests/Feature/LogisticRequestTest.php (hunk 1 above)
Comment (rachadiannovansyah) [reply tests/Feature/LogisticRequestTest.php:291]: woo ggwp sir :smile: okay nope go ahead :rocket:

**Review by rachadiannovansyah — APPROVED** [review]

**Review by sandisunandar99 — APPROVED** [review]
acc

**Review by aldidoanta — APPROVED** [review]

**Pull request merged**